Lab#1.ipynb

```diff
@@ -0,0 +1,283 @@
+    "print(\"from -1z to 1z: \", stats.norm.cdf(1, loc=0, scale=1) - stats.norm.cdf(-1, loc=0, scale=1))\n",
+    "print(\"from -inf to 1z: \", stats.norm.cdf(1, loc=0, scale=1))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Here I integrate the probability density function using the cdf command from scipy; the value 0.34 from the above line is the probability (out of 1) that a data point falls between 0 and 1 standard deviations on a normal distribution with mean 0 and standard deviation of 1. Note that I had to use two cdf commands as the cdf command integrates from the furthest left side of the distribution to the $\\sigma$ value specified, so to get the cumulative probablity from mean to $\\sigma$ instead you must first get the cumulative probability from the far left end of the distribution to $\\sigma$ and then subtract the cumulative probability from the far left end to the mean. That 0.34 probability was cumulative from mean, however I also matched the values from the z-score tables from wikipedia for cumulative probability (0.8413 probability that x is smaller than 1z) and the double sided cumulative-from-mean value (0.683 probability that x is between -1z and 1z)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 58,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Percentile :  0.8413447460685429\n",
+      "Z-score :  1.0\n",
+      "Percentile :  0.15865525393145707\n",
+      "Z-score :  -1.0\n"
+     ]
+    }
+   ],
+   "source": [
+    "q = stats.norm.cdf(1, loc=0, scale=1)\n",
+    "print(\"Percentile : \", q)\n",
+    "print(\"sigma : \", stats.norm.ppf(q, loc=0, scale=1))\n",
+    "q = stats.norm.cdf(-1, loc=0, scale=1)\n",
+    "print(\"Percentile : \", q)\n",
+    "print(\"sigma : \", stats.norm.ppf(q, loc=0, scale=1))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Doing the inverse of the operation above, given the probability/percentile of the data point find the z-score in relation to the mean. Experimented with multiple values, but the easiest to demonstrate was left in the cell above; the 84.13 percentile is the probability that $x$ is less than or equal to one standard deviation to the right from the mean of the normal distribution. The function can also return a negative Z-score if the percentile falls left of the mean; when inputting the percentile 0.159 which corresponds to the cumulative probablity of the distribution up to -1 standard deviations, it returns -1 for the Z-score. Since the normal distribution is symmetric, the probability of getting something below the -1 zscore is the same as getting the probability of something above the +1 zscore; when trying to calculate sigma certainty, one can just find the zscore of 1 - probability of the measured signal instead of having to deal with negative sigmas."
```

Comment:
Your use of the phrase Z-score is inconsistent since sometimes its zscore and sometimes its Z-score and z-score